Cargo.toml

```diff
@@ -87,6 +87,7 @@ codegen-units = 1
 
 [patch.crates-io]
 hdwallet = { git = "https://github.com/nuttycom/hdwallet", rev = "576683b9f2865f1118c309017ff36e01f84420c9" }
+incrementalmerkletree = { git = "https://github.com/nuttycom/incrementalmerkletree", rev = "a858de06d85751926ac0446efae8cb009d7053c4" }
```

Comment:
This is not the latest commit on https://github.com/zcash/incrementalmerkletree/pull/32 . Update when that PR is merged.